assets/wizards/advertising/views/settings/index.js

```diff
@@ -5,20 +5,23 @@
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
```

Comment:
Doesn't seem like this is actually used in this module?

Reply:
You're right! I tend to miss those because the linter let me know, I wonder why it didn't this time 🤔 
Fixed by 4c7267cb8d9f03642cafe5fcfa65c091b3d485fa